### PR TITLE
More Skyrim docs fixes

### PR DESCRIPTION
**Pull request by JonathanFeenstra**
* Change SSEEdit to TES5Edit in LE docs
* Fix local appdata path in SE docs
* Use USSEP as example instead of LE-version in SE docs

### CODE REVIEW

**Review by Sewer56 — APPROVED** [review]
LGTM, I'll let someone else look before merging.

**Review by halgari — APPROVED** [review]

**Pull request merged**
